frontend/src/app/commonComponents/ProtectedRoute.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
   path: string;
   component?: React.FC<any>;
   render?: any;
+  exact?: boolean;
```

Comment:
Does this prop need to be passed to the `Route` component below?

Reply:
yes, without exact it matches /results with /results/:page and vice versa